components/motor/gpio/mathutils.go

```diff
@@ -34,11 +32,6 @@ func goForMath(maxRPM, rpm, revolutions float64) (float64, time.Duration) {
 		rpm = -1 * maxRPM
 	}
 
-	if revolutions == 0 {
-		powerPct := rpm / maxRPM
-		return powerPct, 0
-	}
-
 	dir := rpm * revolutions / math.Abs(revolutions*rpm)
```

Comment:
There's a divide by zero case that we need to test for now returning from goForMath.

Reply:
oops missed a lot

Reply:
Yeah.... run `make test-go` locally or `go test` in the folders you've edited so you realize before you get to the PR phase, it always helps and I see a lot of the math logic is failing, so that's great!